src/AirshipBindings.Android.Core/packages.config

```diff
@@ -1,23 +1,24 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
-  <package id="Xamarin.AndroidX.Activity" version="1.0.0" targetFramework="monoandroid90" />
+  <package id="Xamarin.AndroidX.Activity" version="1.1.0" targetFramework="monoandroid10.0" />
```

Comment:
This mixing of target frameworks worries me. I'd feel much better if they were all consistent.